### PR TITLE
Add captcha to registration view

### PR DESCRIPTION
**Pull request by kbeker**
No issue for that. It is security feature for registration.

### CODE REVIEW

File: users/forms.py [hunk 1]
```diff
@@ -67,7 +68,8 @@ class Meta:
 
 class CustomUserSignUpForm(UserCreationForm):
     email = forms.EmailField(max_length=constants.EMAIL_MAX_LENGTH, required=True, label="id_email")
+    captcha = CaptchaField()
 
     class Meta:
         model = CustomUser
-        fields = ("email", "password1", "password2")
+        fields = ("email", "password1", "password2", "captcha")
```
Comment (maciejSamerdak) [inline users/forms.py]: If I'm not mistaken, this should contain all the fields that we want to import from the source model. So why is `captcha` mentioned here? It isn't a model field, is it? 😕

File: users/static/users/scripts/captcha-refresh.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+$(function () {
+    $("#captcha-refresh").click(function()  {
+        $.getJSON("/captcha/refresh/", {}, function(json) {
+            document.getElementsByClassName('captcha')[0].attributes.src.value = json.image_url;
+            document.getElementById("id_captcha_0").value = json.key;
+        });
+        return false;
+    });
+});
```
Comment (maciejSamerdak) [inline users/static/users/scripts/captcha-refresh.js]: Was this the source of that major frustration of yours that you manifested so verbally last Friday? 😆

File: users/forms.py (hunk 1 above)
Comment (rwrzesien) [reply users/forms.py]: Also non-model fields should be listed here.

File: users/static/users/scripts/captcha-refresh.js (hunk 2 above)
Comment (rwrzesien) [reply users/static/users/scripts/captcha-refresh.js]: @kbeker You are mixing jQuery style with vanilla Javascript style here, works for me but if you want you could unify this :)

Comment (kbeker) [reply users/static/users/scripts/captcha-refresh.js]: @maciejSamerdak exactly that was it. It didn't want to cooperate with me last friday :P @rwrzesien when I was doing this it was driving me crazy so when it finally started to work I just left it and did not touch :D But I'll try to unify this :)

Comment (kbeker) [reply users/static/users/scripts/captcha-refresh.js]: @rwrzesien oh, now it did work from first shot. Changed completely to `jQuery`.

**Pull request merged**